diff --git a/src/schema.rs b/src/schema.rs
index 38efc99..043e5aa 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -14,16 +14,16 @@ diesel::table! {
     }
 }
 
 diesel::table! {
     clock_stubs (id) {
         id -> Integer,
-        start_time -> Text,
-        end_time -> Nullable<Text>,
+        start_time -> TimestamptzSqlite,
+        end_time -> Nullable<TimestamptzSqlite>,
         user_id -> BigInt,
-        last_interaction -> Text,
+        last_interaction -> TimestamptzSqlite,
         afk_warn_flag -> Bool,
     }
 }
 
 diesel::table! {
     configs (key) {
@@ -35,24 +35,24 @@ diesel::table! {
     }
 }
 
 diesel::table! {
     reservations (id) {
         id -> Integer,
-        start_time -> Text,
-        end_time -> Text,
+        start_time -> TimestamptzSqlite,
+        end_time -> TimestamptzSqlite,
         alerted -> Bool,
         user_id -> BigInt,
     }
 }
 
 diesel::table! {
     transactions (id) {
         id -> Integer,
         clock_stub_id -> Integer,
-        buyOrderTime -> Text,
+        buyOrderTime -> TimestamptzSqlite,
         buyOrderIds -> Text,
         buyReady -> Bool,
         buyAvgPrice -> Nullable<Double>,
         sellOrderIds -> Text,
         sellReady -> Bool,
         sellAvgPrice -> Nullable<Double>,
